pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = settings
+addopts = --cov . --cov-config .coveragerc --cov-report term --cov-report html
```

Comment:
I think `.coveragerc` is the default value for `--cov-config`, did you hit a problem that required explicitly specifying it?  Also, we normally use `term-missing` instead of `term`.